frontend/src/pages/application/routes.js

```diff
@@ -27,10 +27,8 @@ import ApplicationIndex from './index.vue'
 
 export default [
     {
-        path: '/application/:id',
-        redirect: to => {
-            return `/application/${to.params.id}/instances`
-        },
+        path: ':id',
```

Comment:
because we moved these routes under the team namespace we don't need to define it as a root route (leading /) and leaving application in would have resulted in a route with the following form `/team/<team-slug>/applications/application/<application-id>/device-groups` which didn't make sense